__init__.py

```diff
@@ -743,6 +744,21 @@ def startSearch(self, value):
     queryText = bpy.context.scene.thangs_model_search
     fetcher.search(query=queryText)
 
+
+def uninstall_old_version_timer():
+    def is_old_addon(mod):
+        name = mod.bl_info['name']
+        if name == 'Thangs Model Search':
+            if mod.__name__ == 'thangs-breeze' and 'RandyHucker' in json.dumps(mod.bl_info):

```

Comment:
Various versions of the older addon have links to the repository under various fields, so we just dump the whole thing to a json string and search for the old repository name in there.

We need to add this check because in the case of automatic updates, the latest addon could still be in a thangs-breeze module.